src/WarcraftLegacies.Source/Quests/Cthun/QuestFiendThousandFaces.cs

```diff
@@ -13,17 +13,17 @@ public sealed class QuestFiendThousandFaces : QuestData
   {
 
     /// <inheritdoc/>
-    public override string RewardFlavour => "Yogg-saron has been consumed. His genome has been added to the brood!";
+    public override string RewardFlavour => "Yogg-Saron has been consumed. His genome has been added to the brood!";
 
     /// <inheritdoc/>
     protected override string RewardDescription =>
-      $"Soldier and Super Major gain the Vampiric Attack ability";
+      $"Soldier and Super Major gain the Vampiric Attack ability.";
```

Comment:
`RewardDescription` should not end with punctuation!
```suggestion
      $"Soldier and Super Major gain the Vampiric Attack ability";
```